assets/css/style.css

```diff
@@ -3,40 +3,78 @@
   src: url(../fonts/yeseva-regular.ttf);
 }
 
+.about-cook {
+  transform: translateX(15%);
+}
+
+.about-food {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  max-width: 430px;
+  text-align: center;
```

Comment:
I think it's redundant

Reply:
idk how to place it the other way

Reply:
I need to see the deploy to help you

Reply:
http://lambda.ingredients-section.surge.sh/